kore/src/Kore/Builtin/Builtin.hs

```diff
@@ -131,6 +133,39 @@ import qualified Kore.Unification.Unify as Monad.Unify
     )
 import Kore.Unparser
 
+
+kreflectionSort :: Text
+kreflectionSort = "KREFLECTION.kreflection"
+
+assertSort :: SortVerifier
+assertSort = verifySort kreflectionSort
+
+verifiers :: Verifiers
+verifiers =
+    Verifiers
+        { sortDeclVerifiers = mempty
+        , symbolVerifiers = symbolVerifiersKreflection
+        , patternVerifierHook = mempty
+        }
+
+symbolVerifiersKreflection :: SymbolVerifiers
+symbolVerifiersKreflection =
+    HashMap.fromList
+    [ ( "KREFLECTION.isConcrete"
+      , verifySymbolKreflection assertSort [assertSort, assertSort]
+      )
+    ]
+
+verifySymbolKreflection
+    :: SortVerifier  -- ^ Builtin result sort
+    -> [SortVerifier]  -- ^ Builtin argument sorts
+    -> SymbolVerifier
+verifySymbolKreflection _ _ =
```

Comment:
I suggest renaming this to `rejectSymbolIsConcrete`. It should not take any arguments, and it should not be necessary to define `kreflectionSort` or `assertSort`.